internal/services/notification/runevents.go

```diff
@@ -115,6 +131,26 @@ func (n *NotificationService) runEventsHandler(ctx context.Context) error {
 				n.log.Error().Msgf("run event %q is not valid", ev.RunEventType)
 			}
 
+			err := n.d.Do(ctx, func(tx *sql.Tx) error {
```

Comment:
Handle webhooks should check that the event sequence has not already been processed.

Reply:
@sgotti do you mean to add a foreint key to RunWebhook(or RunWebhookDelivery) table? it can be RunEvent id or RunEvent sequence.
Or do you mean the same or similar check as in the previous comment?

Reply:
The final idea is to avoid generating double webhooks if we process the run events (create the webhooks) and then saving the last processed sequence fail. We could handle this in different way.

One solution is to make all of this inside a single transaction (so the webhooks creations and the saving of the sequence are atomic). For this reason in the previous PR I asked to split the webhook creation and the delivery.